app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -488,8 +479,8 @@ private void showFeedInformation(final Feed feed, Map<String, String> alternateF
             }
             ArrayAdapter<String> adapter = new ArrayAdapter<>(this, android.R.layout.simple_spinner_item, alternateUrlsTitleList);
             adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
-            spAlternateUrls.setAdapter(adapter);
-            spAlternateUrls.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            binding.spinnerAlternateUrls.setAdapter(adapter);
+            binding.spinnerAlternateUrls.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
```

Comment:
What do you think about `alternateUrlsSpinner`?

Reply:
> Sorry for complaining so much
I understand.
I have the same doubts regarding this too. It's long and also a bit confusing too (for me). In this case, we have to add prefix/suffix for a spinner, otherwise, we may not be able to understand what it stands for. Now, if we add suffix/prefix here, we may need to add that for some other views too. like, your 2nd comment, It seems to add some prefix or suffix too.